crates/mun_abi/src/lib.rs

```diff
@@ -25,6 +25,15 @@ pub mod prelude {
     };
 }
 
+/// Defines the current abi version
+pub const ABI_VERSION: i32 = 00_20_00;
```

Comment:
Did you have a reason in mind for choosing a signed integer (`i32`) over an unsigned integer (`u32`)?

Reply:
Not really, should i change it to a u32?

Reply:
I don't think we'll ever hit the maximum number of versions possible with `i32`, but still I think that `u32` would be more correct - as I don't think we'll ever use negative version numbers.